polaris-react/src/components/Pagination/Pagination.scss

```
@@ -0,0 +1,20 @@
+@import '../../styles/common';
+
+.Pagination {
+  button {
+    #{$se23} & {
+      /* stylelint-disable-next-line declaration-no-important -- se23 */
+      border: none !important;
```

Should we also override box-shadow coming from button to match Figma? Not sure if we decided to keep pagination buttons with box-shadow.

```suggestion
      border: none !important;
      box-shadow: none !important;
```

---

Yes 💯 I was looking at the details page where header pagination buttons have no shadow